# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/cocoa/nsChildView.mm
# Commit: 936591fa4a69
# Full Hash: 936591fa4a69557098f1ac1892211f78c005c9c6
# Author: Edgar Chen <echen@mozilla.com>
# Date: 2021-01-05 21:56:58
# Regressor Bug: 1653949
# File Overlap Count: 1
# Description:
#   Bug 1662363 - Rename eTopLevel/eChild to ePlatformTopLevel/ePlatformChild; r=smaug
#   
#   This is a follow-up of bug 1653949.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D100729
# ==============================================================================

diff -r ff581d36e640 -r 936591fa4a69 widget/cocoa/nsChildView.mm
--- a/widget/cocoa/nsChildView.mm	Tue Jan 05 09:47:53 2021 +0000
+++ b/widget/cocoa/nsChildView.mm	Tue Jan 05 09:48:24 2021 +0000
@@ -5013,11 +5013,11 @@
   if (sLastMouseEventView != oldView) {
     // Send enter and / or exit events.
     WidgetMouseEvent::ExitFrom exitFrom = [sLastMouseEventView window] == [oldView window]
-                                              ? WidgetMouseEvent::eChild
-                                              : WidgetMouseEvent::eTopLevel;
+                                              ? WidgetMouseEvent::ePlatformChild
+                                              : WidgetMouseEvent::ePlatformTopLevel;
     [oldView sendMouseEnterOrExitEvent:aEvent enter:NO exitFrom:exitFrom];
     // After the cursor exits the window set it to a visible regular arrow cursor.
-    if (exitFrom == WidgetMouseEvent::eTopLevel) {
+    if (exitFrom == WidgetMouseEvent::ePlatformTopLevel) {
       [[nsCursorManager sharedInstance] setCursor:eCursor_standard];
     }
     [sLastMouseEventView sendMouseEnterOrExitEvent:aEvent enter:YES exitFrom:exitFrom];